bundles/org.openhab.binding.bluetooth.bluez/pom.xml

```diff
@@ -15,18 +15,21 @@
   <name>openHAB Add-ons :: Bundles :: BlueZ Bluetooth Adapter</name>
 
   <dependencies>
+
     <dependency>
       <groupId>org.openhab.addons.bundles</groupId>
       <artifactId>org.openhab.binding.bluetooth</artifactId>
       <version>${project.version}</version>
       <scope>provided</scope>
     </dependency>
+
     <dependency>
-      <groupId>org.openhab.osgiify</groupId>
-      <artifactId>intel-iot-devkit.tinyb</artifactId>
-      <version>0.5.1</version>
-      <scope>compile</scope>
+      <groupId>com.github.hypfvieh</groupId>
+      <artifactId>bluez-dbus-osgi</artifactId>
+      <version>0.1.3</version>
+      <scope>provided</scope>
```

Comment:
Is this used by any other bundle or available from a core-bom? Otherwise change scope to `compile` and remove it from the feature.

Reply:
No, this will only be used by this add-on here.

Reply:
I switched it over to use `compile` and removed it from both the feature.xml and the footer.xml. But then I get this build error:
```
Feature resolution failed for [openhab-binding-bluetooth-bluez/3.0.0.SNAPSHOT]
[ERROR] Message: Unable to resolve root: missing requirement [root] osgi.identity; osgi.identity=openhab-binding-bluetooth-bluez; type=karaf.feature; version=3.0.0.SNAPSHOT; filter:="(&(osgi.identity=openhab-binding-bluetooth-bluez)(type=karaf.feature)(version>=3.0.0.SNAPSHOT))" [caused by: Unable to resolve openhab-binding-bluetooth-bluez/3.0.0.SNAPSHOT: missing requirement [openhab-binding-bluetooth-bluez/3.0.0.SNAPSHOT] osgi.identity; osgi.identity=org.openhab.binding.bluetooth.bluez; type=osgi.bundle; version="[3.0.0.202010221615,3.0.0.202010221615]"; resolution:=mandatory [caused by: Unable to resolve org.openhab.binding.bluetooth.bluez/3.0.0.202010221615: missing requirement [org.openhab.binding.bluetooth.bluez/3.0.0.202010221615] osgi.wiring.package; filter:="(&(osgi.wiring.package=jnr.constants.platform)(version>=0.9.0)(!(version>=1.0.0)))"]]
```
Any ideas? All the jnr jars are embedded within the bluez-dbus-osgi jar from what I can tell so I don't know why it says they are missing. `jnr.constants.platform` isn't even among the package imports of the bluez-dbus-osgi jar.

Reply:
No idea, really. Maybe @J-N-K?